src/test-utils/rest-api-mocking.ts

```diff
@@ -0,0 +1,6 @@
+import { createAPIMockingUtility } from '@lifeomic/one-query/test-utils';
+import { RestAPIEndpoints } from '../types/rest-types';
+
+export const useRESTAPIMocking = createAPIMockingUtility<RestAPIEndpoints>({
+  baseUrl: 'https://api.us.lifeomic.com',
```

Comment:
Whether in this PR or a future one, let's make this a default, and allow for `apiBaseUrl` from `DeveloperConfig` override it.

```
const { apiBaseURL } = useDeveloperConfig();
...
baseUrl: apiBaseURL ?? 'https://api.us.lifeomic.com',
```

Reply:
Sure, I'm open to trying to inherit it here somehow. I'll keep an eye on for a future PR.